integration/GET_KERNEL_QEMU

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
```

Comment:
set -e

Reply:
I had that but this is intended to be sourced, and you don't want to set -e in a file you source. It's a fast way to exit your login shell if run interactively and anything goes wrong. Scripts that call this can set it.